### PR TITLE
[wip] Add name of module in user-agent in request.

### PR DESCRIPTION
**Pull request by ddurieux**
Add info who make the request in DEBUG mode in the user-agent

### CODE REVIEW

File: alignak/http/client.py [hunk 1]
```diff
@@ -156,7 +157,12 @@ def get(self, path, args={}, wait='short'):
         uri = self.make_uri(path)
         timeout = self.make_timeout(wait)
         try:
-            rsp = self._requests_con.get(uri, params=args, timeout=timeout, verify=self.strong_ssl)
+            headers = {
+                'content-type': 'application/zlib',
```
Comment (Seb-Solon) [inline alignak/http/client.py]: We may have talked about it but are you sure it's always in zlib format? Maybe  this statement force the zlib format?

File: alignak/objects/satellitelink.py [hunk 2]
```diff
@@ -145,7 +145,7 @@ def create_connection(self):
         self.con = HTTPClient(address=self.arb_satmap['address'], port=self.arb_satmap['port'],
                               timeout=self.timeout, data_timeout=self.data_timeout,
                               use_ssl=self.use_ssl,
-                              strong_ssl=self.hard_ssl_name_check
+                              strong_ssl=self.hard_ssl_name_check, name='Arbiter'
```
Comment (Seb-Solon) [inline alignak/objects/satellitelink.py]: The arbiter is the destination here. It's kinda confusing to get the user agent for the destination and not the source.

File: alignak/satellite.py [hunk 3]
```diff
@@ -218,7 +218,7 @@ def do_pynag_con_init(self, s_id):
         try:
             sch_con = sched['con'] = HTTPClient(
                 uri=uri, strong_ssl=sched['hard_ssl_name_check'],
-                timeout=timeout, data_timeout=data_timeout)
+                timeout=timeout, data_timeout=data_timeout, name='Scheduler ' + sname)
```
Comment (Seb-Solon) [inline alignak/satellite.py]: Same here

File: alignak/scheduler.py [hunk 4]
```diff
@@ -984,7 +984,8 @@ def pynag_con_init(self, s_id, s_type='poller'):
 
         uri = links[s_id]['uri']
         try:
-            links[s_id]['con'] = HTTPClient(uri=uri, strong_ssl=links[s_id]['hard_ssl_name_check'])
+            links[s_id]['con'] = HTTPClient(uri=uri, strong_ssl=links[s_id]['hard_ssl_name_check'],
```
Comment (Seb-Solon) [inline alignak/scheduler.py]: Same here

**Comment by ddurieux** [comment]
I close

**Pull request closed without merging**